webapp/src/containers/Rhs/views/LinkedChannels/LinkedChannels.container.tsx

```diff
@@ -43,48 +44,50 @@ export const LinkedChannels = () => {
     }, []);
 
     // Increase the page number by 1
-    const handlePagination = () => {
-        setPaginationQueryParams({...paginationQueryParams, page: paginationQueryParams.page + 1,
-        });
-    };
+    const handlePagination = useCallback(() => {
+        setPaginationQueryParams((prev) => ({...prev, page: paginationQueryParams.page + 1,
```

Comment:
any strong reason not to use `prev.page + 1` instead of `paginationQueryParams.page + 1`?